### ISSUE TITLE
PkByXsl.motive() returns the classpath URL string instead of the markdown content for all XSL lints

### ISSUE THREAD

**Issue by VasilevNStas** [objectionary/lints#1370]
## Problem

`PkByXsl.motive()` — the provider that backs `motive()` of every XSL-based lint — returns a **URL string** instead of the markdown content of the motive file. As a result, calling `motive()` on any of the ~89 XSL lints obtained through `Source`/`PkByXsl` returns garbage like `jar:file:/.../org/eolang/motives/misc/redundant-object.md!/` rather than the actual explanation text.

## Root cause

`src/main/java/org/eolang/lints/PkByXsl.java:74-78` builds the motive input as:

```java
new InputOf(
    PkByXsl.XSL_PATTERN.matcher(
        PkByXsl.LINTS_PATH.matcher(url).replaceAll("eolang/motives")
    ).replaceAll(".md")
)
```

Two independent problems:

1. `InputOf(CharSequence)` in cactoos treats the argument as **content**, not as a classpath path — the string itself is returned verbatim.
2. The derived string still carries the `jar:file:/...!/` prefix, so even with `ResourceOf` it would not resolve.

## Minimal example

```java
final XML xmir = new XMLDocument("<object/>");
final Collection<Defect> defects = new Source(xmir).defects();
// take any XSL lint, e.g. defects().iterator().next()
defects.iterator().next().motive();
// returns "jar:file:/.../org/eolang/motives/misc/...md!/..." instead of markdown
```

## Expected behavior

`motive()` returns the markdown content of `org/eolang/motives/<dir>/<name>.md`, the same way the `LtByXsl(String)` constructor does via `ResourceOf`.

## Why tests didn't catch it

- `PkByXslTest.checksAllMotives` only asserts non-empty.
- `LtByXslTest.returnsMotive` uses the other constructor (`ResourceOf`), which works correctly.

## Related

- #1277 (Java lints from YAML packs) resolves motives through the same broken path.

**Comment by 0crat** [comment]
@vasilevnstas Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+650](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
